modules/renter/hostdb/hostentry.go

```diff
@@ -120,7 +120,7 @@ func (hdb *HostDB) IsOffline(addr modules.NetAddress) bool {
 		return false
 	}
 	if h, ok := hdb.allHosts[addr]; ok {
-		return !h.online
+		return !h.Online
 	}
 	// no record of the host; add it to the HostDB
 	hdb.insertHost(modules.HostSettings{NetAddress: addr})
```

Comment:
If the renter checks a host once, it's probably going to check it again later. So it makes sense to add it to the hostdb at some point. I agree that this is a little implicit though.

`insertHost` is already non-blocking because it calls `scanHost`, which spawns a goroutine to send down a channel. Besides, since `insertHost` requires a lock, we'd need to re-lock the mutex inside this new goroutine. Bleh.